src/net.md

```diff
@@ -1070,6 +1152,7 @@ fn run() -> Result<()> {
 [`reqwest::RequestBuilder`]: https://docs.rs/reqwest/0.6.2/reqwest/struct.RequestBuilder.html
 [`reqwest::Response`]: https://docs.rs/reqwest/*/reqwest/struct.Response.html
 [`reqwest::get`]: https://docs.rs/reqwest/*/reqwest/fn.get.html
+[`reqwest::header::Range`]: https://docs.rs/reqwest/0.7.3/reqwest/header/enum.Range.html
```

Comment:
please use wildcard version `*` in docs.rs links (please also update any other links with hardcoded version that you find in this file)